doc/basics/dependencies.md

```diff
@@ -90,40 +92,43 @@ To this end, just provide the corresponding '*_DIR' to cmake at configuration ti
 Currently supported (note that these paths must refer to the installation directory of the corresponding library):
 [//]: # (generated running ../script/list_dep.py from Radium-Engine/external directory)
 
- * `assimp_DIR`
- * `tinyply_DIR`
- * `glm_DIR`
- * `glbinding_DIR`
- * `globjects_DIR`
- * `stb_DIR`
- * `Eigen3_DIR`
- * `OpenMesh_DIR`
- * `cpplocate_DIR`
- * `nlohmann_json_DIR`
+ *  `assimp_DIR`
+ *  `tinyply_DIR`
+ *  `glm_DIR`
+ *  `glbinding_DIR`
+ *  `globjects_DIR`
+ *  `stb_DIR`
+ *  `tinyEXR_DIR`
+ *  `Eigen3_DIR`
+ *  `OpenMesh_DIR`
+ *  `cpplocate_DIR`
+ *  `nlohmann_json_DIR`
```

Comment:
Are these changes intentional ? I can't find the reason why spaces are added (same for the following lines)